packages/caver-core-method/src/index.js

```diff
@@ -286,18 +286,27 @@ const buildSendSignedTxFunc = (method, payload, sendTxCallback) => (sign) => {
 }
 
 const buildSendRequestFunc = (defer, sendSignedTx, sendTxCallback) => (payload, method) => {
+  // Logic for handling multiple cases of parameters in sendSignedTransaction.
+  // 1. Object containing rawTransaction
```

Comment:
Please describe what happens for this case.

Reply:
Comments are added :)